deploy/strategies/deployEraStrategies.ts

```diff
@@ -13,17 +13,15 @@ const deployContractUsingFactoryWithBytecodeOnEra = async (
   constructorArgs?: Args,
 ): Promise<string> => {
 
-  // const provider = new Provider("https://sepolia.era.zksync.dev");
-
-  // const deployerAddress = new Wallet(process.env.DEPLOYER_PRIVATE_KEY as string, provider);
+  const deployerAddress = new Wallet(process.env.DEPLOYER_PRIVATE_KEY as string);
 
-  // const ContractArtifact = await hre.artifacts.readArtifact("ContractFactory");
+  const ContractArtifact = await hre.artifacts.readArtifact("ContractFactory");
 
-  // const deployerContract = new ethers.Contract(
-  //   contractFactoryAddress,
-  //   ContractArtifact.abi,
-  //   deployerAddress
-  // );
+  const deployerContract = new ethers.Contract(
+    contractFactoryAddress,
+    ContractArtifact.abi,
+    deployerAddress
+  );
```

Comment:
^ this fails with `TypeError: Cannot read properties of undefined (reading 'Contract')`

Reply:
Hello, we checked it out, and it seems that you want to use `ethers` from `hardhat` but you didn't imported it in the `era.hardhat.config` file. 
```javascript
import "@nomicfoundation/hardhat-ethers"
```
Also, I would recommend to use our plugins (and SDK) in that script for communicating with the zkSync networks.